netlify.toml

```diff
@@ -0,0 +1,9 @@
+[Settings]
+
+ID = "44effe10-3635-426d-899d-984201fa828a"
```

Comment:
What is this `ID` used for ?
It should not be in Circle CI env vars ?

Reply:
> [Settings]
> //Added automatically by the Netlify CLI. It has no effect during normal git-backed deploys.
> ID = "Your_Site_ID"

Source: https://www.netlify.com/docs/netlify-toml-reference/

It seems safe since you need to login to deploy, but I could be wrong

Reply:
I remove this configuration because it was useless. The  ID will be stored in a CircleCi variable `NETLIFY_SITE_ID`.

This is related to the documentation : https://www.netlify.com/docs/cli/#linking-and-unlinking-sites